docs/root/start/start.rst

```diff
@@ -0,0 +1,154 @@
+.. _start:
+
+Getting Started
+===============
+
+This section gets you started with a very simple configuration, and provides some example configurations.
```

Comment:
Prefer no comma in this situation (doesn't seem to be Oxford comma as there aren't 3 items).